src/SFA.DAS.Rofjaa.Application/Agencies/Services/AgenciesService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using SFA.DAS.Rofjaa.Domain.Entities;
+using SFA.DAS.Rofjaa.Domain.Interfaces;
+
+namespace SFA.DAS.Rofjaa.Application.Agencies.Services
+{
+    public class AgenciesService : IAgencyService
```

Comment:
I could be wrong, but I think we no longer need this service or repo.

Reply:
Agreed! Removed:

IAgencyService.cs
AgenciesSercice.cs

Updated AddServiceRegistrationExtension.cs to not fire them.